packages/cart/src/models/adjustment-line.ts

```diff
@@ -16,12 +16,6 @@ export default abstract class AdjustmentLine {
   @Property({ columnType: "text", nullable: true })
   description: string | null = null
 
-  @Property({
-    columnType: "text",
-    nullable: true,
-  })
-  promotion_id: string | null = null
-
```

Comment:
is that intended? feels a bit out of scope no?

Reply:
It is intended, as we need separate indexes for the inheriting classes